jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerCredentialHelper.java

```diff
@@ -65,7 +63,7 @@ public DockerCredentialHelper(String serverUrl, Path credentialHelper) {
   }
 
   DockerCredentialHelper(String registry, String credentialHelperSuffix) {
-    this(registry, Paths.get(CREDENTIAL_HELPER_PREFIX + credentialHelperSuffix));
+    this(registry, Paths.get("docker-credential-" + credentialHelperSuffix));
```

Comment:
I wonder if we should get rid of this method and push knowledge of the `docker-credential-` prefix to the callers/creators?

Reply:
Let me take care of this in a separate PR.